samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -69,7 +62,10 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
         const pageTitle = convertPageStateToString(state);
         document.title = `${pageTitle} - ${webAppTitle}`;
 
-        callIdRef.current = state?.call?.id;
+        if (!callIdRef.current && state?.call?.id) {
```

Comment:
note: this may need to also check if `callIdRef !== state?.call?.id` to log a new callId if the callId changes

Reply:
After my testing, I found that it outputs a new callid when you join, exit and rejoin, or join a new session after exiting.

But just in case, I revised the code to add your note for edge cases that were not tested. I tested the new edit and the logs are outputting as expected. New callids when joining, then rejoining a call.

Reply:
Yeah the call ID may change _during_ an active call - I'm not clear when this happens, something very back-end related.